sbi/simulators/simutils.py

```diff
@@ -117,56 +117,59 @@ def simulate_in_batches(
     parameter_sample_fn: Callable,
     num_samples: int,
     simulation_batch_size: int,
-    x_dim: torch.Size,
```

Comment:
If it is a shape, shouldn't it be called a shape? (or, annoyingly, a size in PyTorch?) 
This could be a better name `output_shape`. Quoting from somewhere:

> A numpy array is a grid of values, all of the same type, and is indexed by a tuple of nonnegative integers. The number of dimensions is the rank of the array; the shape of an array is a tuple of integers giving the size of the array along each dimension.

(rank unrelated to the linear algebra concept).

I believe @janfb and @michaeldeistler we could benefit from discussing consistent use of the numel, shape, size, ndim terminology across the codebase.

Reply:
In this specific instance, it's not in the code anymore anyway (I removed the line in this commit). But, generally, I agree and we can discuss.